pages/components/Lists/list.css

```diff
@@ -0,0 +1,77 @@
+@import url(/css/main.css);
+
+.lists-preview-container {
```

Comment:
suggestion- put all css related to flex at one place.
E.g: 
```css
    display: flex;
    justify-content: space-evenly;
    align-items: center;
    gap: 1rem;
    flex-wrap: wrap;

    height: fit-content;
    border: 1px solid var(--secondary-faded-color);
    border-radius: 8px;
    padding: 1rem;
```
This will increase readability also your code will look neat and clean.

Reply:
Done, thanks for the suggestion